docs/core/compatibility/windows-forms/10.0/obsolete-apis.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Breaking change: .NET 10 obsoletions and warnings in Windows Forms"
+titleSuffix: ""
+description: Learn about the .NET 10 breaking change where some Windows Forms APIs have been marked as obsolete or otherwise produce a warning.
+ms.date: 03/10/2025
+---
+# Windows Forms obsoletions and warnings (.NET 10)
+
+Some Windows Forms APIs have been marked as obsolete, starting in .NET 10. Other APIs aren't obsolete but will cause a compile-time warning if you reference them.
+
+## Previous behavior
+
+In previous .NET versions, these APIs can be used without any build warning.
+
+## New behavior
+
+In .NET 10 and later versions, use of these APIs produces a compile-time warning or error with a custom diagnostic ID. The use of custom diagnostic IDs allows you to suppress the warnings individually instead of blanket-suppressing all obsoletion warnings.
```

Comment:
```suggestion
In .NET 10 and later versions, use of these APIs produces a compile-time warning with a custom diagnostic ID. The use of custom diagnostic IDs allows you to suppress the warnings individually instead of blanket-suppressing all obsoletion warnings.
```